client/blocks/reader-post-card/style.scss

```diff
@@ -653,27 +662,6 @@
 	position: relative;
```

Comment:
It looks like `.reader-post-card__gallery` element is no longer active.

Reply:
Also there is an unused rule targeting `.reader-post-card__gallery-item` in https://github.com/Automattic/wp-calypso/blob/trunk/client/reader/search-stream/style.scss#L360